server/controllers/get-articles.controllers.js

```diff
@@ -0,0 +1,34 @@
+import models from '../models';
+
+const { Article } = models;
+
+const getArticles = async (req, res) => {
+  const { page } = req.params;
+  const perPage = 10;
+
+  const offset = perPage * page - perPage;
```

Comment:
Add a comment that describes this operation

Reply:
I have added the description